src/ucp/rndv/rndv_mtype.inl

```diff
@@ -64,11 +64,31 @@ ucp_proto_rndv_mtype_get_memh(ucp_request_t *req, ucp_md_index_t md_index)
     return mdesc->memh->uct[md_index];
 }
 
+static ucp_ep_h ucp_proto_rndv_mtype_ep(ucp_worker_t *worker,
+                                        ucs_memory_type_t buf_mem_type)
+{
+    ucs_memory_type_t frag_mem_type =
+                                 worker->context->config.ext.rndv_frag_mem_type;
+
+    if (worker->mem_type_ep[buf_mem_type] != NULL) {
+        return worker->mem_type_ep[buf_mem_type];
+    }
+
+    ucs_assert_always(worker->mem_type_ep[frag_mem_type] != NULL);
+
+    return worker->mem_type_ep[frag_mem_type];
```

Comment:
not sure it's a correct check. we should see if there is a memtype ep that can register both buf_mem_type and rndv_frag_mem_type (one for local zcopy, the other for remote read/write)

Reply:
currently we assume that
- we do not need an rkey for memtype_copy, see [ucp_proto_rndv_mtype_copy](https://github.com/openucx/ucx/blob/master/src/ucp/rndv/rndv_mtype.inl#L135)
- if memtype ep exists, it supports put_zcopy/get_zcopy

p.s. seems like existing checks in `ucp_proto_common_find_lanes` are wrong for ppln protocols, but thet is better to address in the other PR

Reply:
ok